src/main/java/org/folio/helper/PurchaseOrderHelper.java

```diff
@@ -512,6 +509,62 @@ private CompletableFuture<Void> setPoNumberIfMissing(CompositePurchaseOrder comp
     return completedFuture(null);
   }
 
+  private void setPoNumberPrefix(CompositePurchaseOrder compPO) {
```

Comment:
Better to remove these helper methods to another util class.

Reply:
fixed.